Xamarin.Forms.Platform.iOS/Cells/ImageCellRenderer.cs

```diff
@@ -22,8 +22,8 @@ public override UITableViewCell GetCell(Cell item, UITableViewCell reusableCell,
 
 		protected override void HandlePropertyChanged(object sender, PropertyChangedEventArgs args)
```

Comment:
I was thinking renaming the  override was a good way to go because the type on the sender is now different. 

Now if someone is overriding this and casting sender they are going to get an Invalid Cast Exception at run time.

What about just renaming this one to how it was HandleCellPropertyChanged

And then from there you can call HandlePropertyChanged with the expected sender that way anyone that has overridden *HandlePropertyChanged* will still get the class they expect